.github/workflows/backend-cli-test.yml

```diff
@@ -3,10 +3,10 @@ name: Backend CLI Test
 on:
   push:
     branches:
-      - master
+      - v2.3
```

Comment:
2.x is acceptable :)

Reply:
both the master and v2.x need to be checked IMO

Reply:
Thanks for reviewing. IMHO, CI for `v2.3` will no longer exist after version `2.3` is released, cause it is not a LTS version. And this change will also not be backported into master branch.  For example, for the release version `2.4`, we can simply use `v2.4` here. Meanwhile, the `master` is the only branch name needed to listed here in the `master` branch.

Once we have LTS versions, then we can keep `master` and all LTS version branches in the `master` branch. So What do you think? 😄 Much appreciated.

Reply:
got it, let's merge

Reply:
> both the master and v2.x need to be checked IMO

 The codes in master branch will also be checked if we commit a PR targeted to `master` branch. So we can place different branch names here, respectively. Please correct me if I was wrong. Thanks.